test/test_core.py

```diff
@@ -6722,6 +6722,7 @@ def test_bullet(self, use_cmake):
   @needs_make('depends on freetype')
   @no_4gb('runs out of memory')
   @is_slow_test
+  @crossplatform
```

Comment:
I'm not sure we actually want to run this test as crossplatform because it's so slow, but I wanted to at least test on CI.
It will get run on emscripten-releases CI.